### ISSUE TITLE
The loss does not drop when I run the program on the datasets Fdataset and Cdataset

### ISSUE THREAD

**Issue by Artillery-HQ** [storyandwine/LAGCN#19]
When I use your original dataset and the lrssl dataset, the resul is normal.But when I run the program on Fdataset and Cdataset, the AUPR is 0.0005 and the AUC is about 0.5. It's abnormal.
![image](https://github.com/storyandwine/LAGCN/assets/56581202/7ac53087-e951-40af-b5ec-4eecf7d295fa)

![image](https://github.com/storyandwine/LAGCN/assets/56581202/43160a82-aa6c-4cae-996d-37c8599dc1f7)

**Comment by storyandwine** [comment]
Thank you for your interest in our work. Although we did not mention the two datasets you referred to in our paper, we are still happy to assist you.

Here are some suggestions that may be helpful:

1. AI algorithms usually require different hyperparameters for different datasets to achieve good performance. Therefore, you may consider adjusting the hyperparameters first.

2. Different datasets usually have different data distributions, so you may need to analyze the differences between different datasets. For link prediction tasks, you may need to analyze the density of the network first. If the distribution of positive and negative samples is too disparate, you can consider designing different weights for positive and negative samples in the loss function to compensate for the difference. Referring to the loss functions used in other research works is also a possible direction to explore.

If you have any other questions, please feel free to contact us at any time. We wish you all the best!

Best regards,

**Comment by Artillery-HQ** [comment]
Thanks for your reply, I found the reason that why the metrics are unsatisfactory.The similarity matrics in dataset should be normalized by rows.
